# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1383168670 -3600
#      Wed Oct 30 22:31:10 2013 +0100
# Node ID 43a4d80523a1a8544b218b7529d5d59d69a01469
# Parent  a9ba926bc06d176a573a6a7f3d93b0cebcc71935
* de_DE.ts: updated German GUI trnaslation

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -269,22 +269,22 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
         <translation>&amp;Hilfe</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation>&amp;Hilfe zu Schlüsselwort</translation>
+        <translation>&amp;Hilfe zum Schlüsselwort</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation>&amp;Dokumentation zu Schlüsselwort</translation>
+        <translation>&amp;Dokumentation zum Schlüsselwort</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
         <translation>&amp;Suchen und Ersetzen</translation>
     </message>
     <message>
         <location line="-584"/>
@@ -328,17 +328,17 @@ konnte nicht zum Schreiben geöffnet werden:
     <message>
         <location line="+53"/>
         <source>&amp;File</source>
         <translation>&amp;Datei</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
-        <translation>&amp;Editieren</translation>
+        <translation>&amp;Barbeiten</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
         <translation>&amp;Einstellungen</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1280,17 +1280,17 @@ zu: </translation>
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
         <translation>Dokumentation zu</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Edit</source>
-        <translation>Editieren</translation>
+        <translation>Bearbeite</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
         <translation>Markierter Quellcode au&amp;sführen</translation>
     </message>
 </context>
 <context>
